src/components/ContactDetails/ContactDetailsAddNewProp.vue

```diff
@@ -21,38 +21,80 @@
   -->
 
 <template>
-	<div class="property property--without-actions property--last">
-		<!-- title -->
-		<PropertyTitle :icon="'icon-add'" :readable-name="t('contacts', 'Add new property')" />
-
-		<div class="property__row">
-			<div class="property__label" />
-
-			<!-- type selector -->
-			<Multiselect :options="availableProperties"
-				:placeholder="t('contacts', 'Choose property type')"
-				class="property__value"
-				track-by="id"
-				label="name"
-				@input="addProp" />
-		</div>
+	<div>
```

Comment:
wrapping div has only one element. drop the div?

Reply:
I think it's the easiest way to center the popup to the button. Without the div, it is centered to the parent div.

![image](https://user-images.githubusercontent.com/74607597/233320463-a19cb212-96f7-4548-af16-ef1289a4f896.png)